services/base-service/models/sitefarm/template/modified-date.js

```diff
@@ -0,0 +1,42 @@
+template = {
+  id: "modified-date",
+  script: {
+    "lang": "mustache",
+    "source": `{
+      "query": {
+        "bool": {
+          "filter": {
+            "bool": {
+              "must": [
+                {
+                  "bool": {
+                    "must": [
+                      { "exists": { "field": "is-visible" }},
+                      { "term" : {"is-visible": true } }
+                    ]
+                  }
+                }
+                {{#gte_date}}
+                ,{
+                  "range": {
+                    "modified-date": {
+                      "gte": "{{gte_date}}"
+                    }
+                  }
+                }
+                {{/gte_date}}
+                ,{
+                  "terms": {
```

Comment:
This is where we find experts with matching ids.  However, the way it's written now, it will fail if you don't include any epxerts.